en/setup/dev_env_linux_ubuntu.md

```diff
@@ -305,7 +305,7 @@ cmake \
 ## Parrot Bebop
 
 Developers working with the Parrot Bebop should install the RPi Linux Toolchain. Follow the
-description under [Raspberry Pi hardware](https://docs.px4.io/en/flight_controller/raspberry_pi_navio2.html).
+description under [Raspberry Pi hardware](#raspberry-pi-hardware).
 
 Next, install ADB.
 
```

Comment:
Change is good but the I think the replaced link is still useful for developers at this point. How above we add another line at the end?:

> **Note** Additional information for setting up Raspberry Pi [can be found here](https://docs.px4.io/en/flight_controller/raspberry_pi_navio2.html).